a3-live/src/components/TeamInterface.js

```diff
@@ -7,68 +7,88 @@ import FormControl from '@material-ui/core/FormControl';
 import ListItemText from '@material-ui/core/ListItemText';
 import Select from '@material-ui/core/Select';
 import Checkbox from '@material-ui/core/Checkbox';
+import {OutlinedInput} from "@material-ui/core";
+import { makeStyles } from '@material-ui/core/styles';
+import FilledInput from '@material-ui/core/FilledInput';
+import FormHelperText from '@material-ui/core/FormHelperText';
+import Card from "@material-ui/core/Card";
+import CardHeader from "@material-ui/core/CardHeader";
+import CardContent from "@material-ui/core/CardContent";
 
+const useStyles = makeStyles(theme => ({
+    root: {
+        display: 'flex',
+        flexWrap: 'wrap',
+    },
+
+    selectEmpty: {
+        marginTop: theme.spacing(2),
+    },
+    card: {
+        maxWidth: 345,
```

Comment:
This is a good place to add `marginBottom` that you mentioned in your commit to be missing.
```suggestion
        maxWidth: 345,
        marginBottom: theme.spacing(2)
```